apps/src/code-studio/pd/regional_partner_mini_contact/RegionalPartnerMiniContact.jsx

```diff
@@ -100,6 +129,13 @@ export class RegionalPartnerMiniContact extends React.Component {
     }
   };
 
+  roleMap = () => {
+    const roleMap = ROLES.map(v => ({value: v, label: v}));
+    // add friendly empty value option
+    roleMap.unshift({value: '', label: '-'});
+    return roleMap;
+  };
```

Comment:
It might be a little better to define `ROLE_MAP` as a constant right after `ROLES`, above, since the former is purely dependent on the latter and doesn't need to be recomputed every render.